### ISSUE TITLE
Ladybird: drop down UI for <select> is in the wrong screen location

### ISSUE THREAD

**Issue by ojhunt** [SerenityOS/serenity#23120]
The drop down for a `<select>` is placed in the wrong location, as the degree of wrongness seems proportional to the position of the element in the page. That seems to imply the position of the drop down is computed relative to the view, but specified relative to the element, but that's just supposition :D

```
<!doctype HTML>
<html>
    <body>
      <div style="width: 200px; height: 200px;"></div>
      <select>
          <option>bad</option>
      </select>
    </body>
</html>
```

**Comment by bplaat** [comment]
You are right, when using the AppKit chrome with high dpi the select dropdown position is wrong. I will fix this in the near future
